nucleus/metrics/cuboid_metrics.py

```diff
@@ -225,8 +248,8 @@ class CuboidRecall(CuboidMetric):
     def __init__(
         self,
         enforce_label_match: bool = True,
-        iou_threshold: float = 0.0,
-        confidence_threshold: float = 0.0,
+        iou_threshold: Optional[float] = None,
+        confidence_threshold: Optional[float] = None,
```

Comment:
Changes to default params needs to be reflected in `nucleus/validate/eval_functions/available_eval_functions.py` configurations as well since that is the client facing code. The defaults there override the defaults provided here if not reflected.